content/post/formatting-with-style.md

```diff
@@ -0,0 +1,247 @@
+---
+author: m09
+date: 2018-10-12
+title: "Formatting. With Style."
+image: /post/formatting-with-style/teapot.jpg
+description: "Formatting code using existing styles."
+categories: ["technical", "MLonCode"]
+---
+
+Ensuring that a codebase is consistent in style is both hard and
+costly, yet it is extremely important for maintainability and to
+reduce technical debt. Why is it hard and costly?  There are many
+contributing factors but the main ones are certainly:
+
+- teams might not have set up a linting process early enough or at all
+- people handling the codebase might be changing on a regular basis
```

Comment:
also: linters do not cover everything. Besides, they can be hard to set up to their full power or can be too opinionated.